test/spec/spec.controller.js

```diff
@@ -640,6 +640,108 @@ describe('lib/base-controller', () => {
       });
     });
 
+    describe('getErrors', () => {
+
+      let req;
+      let res;
+      let errors;
+
+      beforeEach(() => {
+
+        errors = {
+          key1: {
+            key: 'key1',
+            type: 'required'
+          },
+          key2: {
+            key: 'key2',
+            type: 'maxlength',
+            arguments: [10]
+          },
+          key3: {
+            key: 'key3',
+            type: 'before',
+            arguments: [3, 'years']
+          }
+        };
+
+        req = request();
+        req.form = {};
+        res = response();
+        sinon.stub(controller, 'getErrors').returns(errors);
+        sinon.stub(controller, 'getErrorMessage', e => `error for ${e.key}`);
```

Comment:
I *tend* to use `err` when writing verbose code and `e` when writing terse code (like this example).

However, there is the additional consideration that eslint often (legitimately) complains about variable shadowing if you use `err` in multiple places.

As such, I don't see a particular need to standardise on a particular form, as long as `err` and `e` are only ever used for errors, and not other things.